src/Pretzel.Logic/Liquid/DefaultFilter.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Pretzel.Logic.Liquid
+{
+    public static class DefaultFilter
+    {
+        public static string Default(string input, string @defaultValue)
+        {
+            return !string.IsNullOrWhiteSpace(input) ? input : defaultValue;
```

Comment:
Could you reverse the check?
`string.IsNullOrWhiteSpace(input) ? defaultValue : input;`